ui/app/AppLayouts/Chat/panels/UserListPanel.qml

```diff
@@ -60,7 +61,7 @@ Item {
             model: SortFilterProxyModel {
                 sourceModel: root.usersModel
 
-                proxyRoles: ExpressionRole {
+                proxyRoles: FastExpressionRole {
```

Comment:
I wonder how this works :smile: You also have to set the `expectedRoles` property, sth like:

```qml
expectedRoles: ["localNickname", "ensName", "alias"]
```

That said, I think this should just be flagged as `REQUIRED` in the corresponding `Q_PROPERTY` macro definition, and same for the other new `FastExpressionFoo` items; wdyt @micieslak

Reply:
Probably it doesn't! 😄

Reply:
Ok, indeed.
After adding this:
```qml
expectedRoles: ["localNickname", "ensName", "displayName", "alias"]
```
the UI went back to slow switching between channels. It's better than before, but still noticable.

@caybro since it's a UI SFPM-related thing, could you please pick it up from me?
I'm not sure how to deal with this, you or @micieslak know better!

Reply:
I really have no idea what could be wrong here, I still think it might as well be some backend issue; I'd just go ahead and commit this, the enhanced `FastExpressionRole` is worth having in any case and then @micieslak can have a look when he's back?